### PR TITLE
Adding sync website script

### PR DESCRIPTION
**Pull request by jeremygold**
Please check that my script looks OK!

### CODE REVIEW

File: scripts/sync-website.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+gsutil rsync -R ../nanonauts/ gs://gfx.goldsolutions.co.nz/
```
Comment (Goldini) [inline scripts/sync-website.sh]: I don't think u need this now that you've got cloud build.yaml

Comment (jeremygold) [reply scripts/sync-website.sh]: Gosh, you're right - Well spotted. I'll remove it now.

Comment (Goldini) [reply scripts/sync-website.sh]: I no coz im shmart

Comment (jeremygold) [reply scripts/sync-website.sh]: Cool, I've remove that now.

**Pull request merged**